guides/chain-registration/mainchain/constants.ts

```diff
@@ -0,0 +1,10 @@
+const CHAIN_ID_LENGTH = 4;
+const MIN_CHAIN_NAME_LENGTH = 1;
+const MAX_CHAIN_NAME_LENGTH = 32;
+const BLS_PUBLIC_KEY_LENGTH = 48;
+const NUMBER_ACTIVE_VALIDATORS_MAINCHAIN = 101;
+const MESSAGE_TAG_CHAIN_REG = 'LSK_CHAIN_REGISTRATION_';
+const SC_CHAIN_ID = "03000008";
+const SC_NAME = "sidechain_8";
+
+module.exports = { MESSAGE_TAG_CHAIN_REG, SC_CHAIN_ID, SC_NAME, CHAIN_ID_LENGTH, MIN_CHAIN_NAME_LENGTH, MAX_CHAIN_NAME_LENGTH, BLS_PUBLIC_KEY_LENGTH, NUMBER_ACTIVE_VALIDATORS_MAINCHAIN }
```

Comment:
maybe use export keyword?
```suggestion
export const CHAIN_ID_LENGTH = 4;
export const MIN_CHAIN_NAME_LENGTH = 1;
export const MAX_CHAIN_NAME_LENGTH = 32;
export const BLS_PUBLIC_KEY_LENGTH = 48;
export const NUMBER_ACTIVE_VALIDATORS_MAINCHAIN = 101;
export const MESSAGE_TAG_CHAIN_REG = 'LSK_CHAIN_REGISTRATION_';
export const SC_CHAIN_ID = "03000008";
export const SC_NAME = "sidechain_8";

```